pallets/staking/src/lib.rs

```diff
@@ -1827,10 +1829,16 @@ decl_module! {
             ensure!(Self::era_election_status().is_closed(), Error::<T>::CallNotAllowed);
             let controller = ensure_signed(origin)?;
             let mut ledger = Self::ledger(&controller).ok_or(Error::<T>::NotController)?;
+
             ensure!(
                 ledger.unlocking.len() < MAX_UNLOCKING_CHUNKS,
                 Error::<T>::NoMoreChunks,
             );
+            // check if validator
+            if <Validators<T>>::contains_key(&ledger.stash) {
+                // check that the remaining bond balance is at least equal to minimum bond threshold
+                ensure!(ledger.total - value >= <MinimumBondThreshold<T>>::get(), Error::<T>::InvalidValidatorUnbondAmount);
```

Comment:
Need to make sure the check doesn't underflow.
```suggestion
                ensure!(ledger.total.saturating_sub(value) >= <MinimumBondThreshold<T>>::get(), Error::<T>::InvalidValidatorUnbondAmount);
```

Reply:
Also the check should be done using `ledger.active` not `ledger.total`.  The `total` will not go down until after the unbond hold period.

Validators are required to keep at least 50k POLYX actively bonded.